app/components/autocomplete/autocomplete.js

```diff
@@ -177,8 +184,14 @@ export default class Autocomplete extends PureComponent {
         return maxHeight;
     }
 
+    handleAppsTakeOver = (takeOver) => {
+        if (this.state.appsTakeOver !== takeOver) {
+            this.setState({appsTakeOver: takeOver});
```

Comment:
maybe you don't need this state and you can set the prop for `appsTakeOver` if `state.appCommandCount > 0` avoids updating state and re-rendering